### ISSUE TITLE
<q> should be allowed in <byline>

### ISSUE THREAD

**Issue by joeytakeda** [TEIC/TEI#1858]
In a project, we have decided to use the `<q>` element to denote the presence of quotation marks in the source (we may, at a later stage, use more syntactically precise elements). There is a case where the author's pseudonym is put in "scare quotes":

```
   <byline>(By <name ref="pers:WE">WINIFRED REEVE</name>, "<name ref="pers:WE">ONOTO WATANNA</name>")</byline>

```

Why is `<q>` not allowed here in `<byline>`? 

I note that this is perfectly valid:

```
   <byline>(By <name ref="pers:WE">WINIFRED REEVE</name>, <soCalled><name ref="pers:WE">ONOTO WATANNA</name></soCalled>)</byline>

```

But this is not (using the suggested "soCalled" value for `q/@type`):

```
   <byline>(By <name ref="pers:WE">WINIFRED REEVE</name>, <q type="soCalled"><name ref="pers:WE">ONOTO WATANNA</name></q>)</byline>

```

**Comment by sydb** [comment]
Really good question. I see no good reason off the top of my head. The obvious structural answer is that the content of `<byline>` is model.phrase instead of macro.phraseSeq. That does not mean this is an easy fix, as there may be repercussions to using macro.phraseSeq.

**Comment by sydb** [comment]
Just looked a little more carefully. I think changing the content model of `byline` from
~~~~~
  ( text | model.gLike | model.phrase | docAuthor | model.global )*
~~~~~
to 
~~~~~
  ( docAuthor | macro.phraseSeq )*
~~~~~
would do the trick, and I don’t see any ambiguity problems. This would allow `<cit>`, `<quote>`,  `<floatingText>`, and `<said>` to be children of `<byline>`, in addition to `<q>`.

**Comment by ebeshero** [comment]
At Council F2F, we think this is part of a larger set of issues indicating that we need more availability for `<q>`. We're [opening a new ticket](https://github.com/TEIC/TEI/issues/1918) to investigate changing `<q>`'s class membership.  We're opening a new ticket to investigate this and possible ramifications.

**Comment by ebeshero** [comment]
Council notes that this could be addressed with #1918 but @sydb and @peterstadler  should investigate that together. @sydb could open a new ticket about whether the content of `<byline>` be changed to `macro.phraseSeq`.

**Comment by peterstadler** [comment]
implemented by moving `<q>` to `model.hiLike` in #2008
